pkgs/development/tools/build-managers/scala-cli/default.nix

```diff
@@ -1,5 +1,6 @@
-{ stdenv, coreutils, lib, installShellFiles, zlib, autoPatchelfHook, fetchurl, callPackage }:
+{ stdenv, coreutils, lib, installShellFiles, zlib, autoPatchelfHook, fetchurl, makeWrapper, callPackage, jre }:
```

Comment:
Being a bit pedantic, what about putting one input per line?

Reply:
fine with that :)